actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -253,11 +253,11 @@ def permit!
     #   ActionController::Parameters.new(person: {}).require(:person)
     #   # => ActionController::ParameterMissing: param is missing or the value is empty: person
     def require(key)
-      value = self[key]
-      if value.present? || value == false
-        value
+      if key.is_a?(Array)
+        keys.map &method(:require)
```

Comment:
It would be an easier change if we just do an early return:

```ruby
def require(key)
  return key.map(&method(:require)) if key.is_a?(Array)

  # Rest of the method doesn't need to change at all.
end
```